### PR TITLE
Updated "include-inheritable" Parameters

### PR DESCRIPTION
**Pull request by claasahl**
This change is [Reviewable](https://reviewable.io/reviews/scrive/scrive-apidocs/70)

### CODE REVIEW

**Comment by tombusby** [comment]
`formData`?

`include-inheritable` is passed in on the end of the URL as a `?` query param.

**Comment by claasahl** [comment]
It seems a bit odd to use query parameters for `POST` endpoints like `create`, `update` and `delete`.

From what I can tell, it is fine (i.e. it works) to send the parameter `include-inheritable` in the formData section.

**Comment by tombusby** [comment]
hmm. I have seen APIs before where this is the case, especially since this governs what is returned (i.e. the part that basically mimics a `GET` request after the operation is performed) rather than data which is passed in that causes state change. This would also mean creating an inconsistency between the way this concept is expressed for the `GET` vs other requests.

I'd rather leave this as it is tbh but feel free to ask others for their opinion and I'll happily defer if I'm in the minority.

File: documentation/api/usergroup/deleteusergroupcontactdetails.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ post:
         delete
       required: true
     - name: include-inheritable
-      in: query
+      in: formData
       type: boolean
       description: |
         Append `?include-inheritable` to the URL to see a preview of which
```
Comment (tombusby) [inline documentation/api/usergroup/deleteusergroupcontactdetails.yaml]: If we go this route we'd have to update this text before I can approve the PR

Comment (claasahl) [reply documentation/api/usergroup/deleteusergroupcontactdetails.yaml]: good point, I missed that

Comment (claasahl) [reply documentation/api/usergroup/deleteusergroupcontactdetails.yaml]: done

**Comment by tombusby** [comment]
Yeah I thought it might be slightly more standards compliant to do it the way Claas is saying. I also feel it's more intuitive my way ("just add this to the url, regardless of endpoint").

> I'm surprised that it works though

Yeah the way kontrakcja handles fields doesn't care where they come from.

...

to the business at hand though, I'm gonna abdicate my responsibility to decide on this cos I'm not unbiased. I'll leave it to you @wwwater to hit merge or close and break the tie, I don't mind so much either way.

**Comment by wwwater** [comment]
I'll merge it like this then. My argument would be to keep documentation as standard as possible, because API clients might want to use tools to auto-generate code from it.

**Comment by tombusby** [comment]
> My argument would be to keep documentation as standard as possible

yeah that makes a lot of sense

**Pull request merged**